manual/manual/macros.hva

```diff
@@ -12,13 +12,13 @@
   font-family: 'Fira Sans';
   font-style: normal;
   font-weight: 400;
-  src: url('/pub/docs/manual-ocaml/fonts/fira-sans-v8-latin-regular.eot'); /* IE9 Compat Modes */
+  src: url('fonts/fira-sans-v8-latin-regular.eot'); /* IE9 Compat Modes */
   src: local('Fira Sans Regular'), local('FiraSans-Regular'),
-       url('/pub/docs/manual-ocaml/fonts/fira-sans-v8-latin-regular.eot?\#iefix') format('embedded-opentype'), /* IE6-IE8 */
-       url('/pub/docs/manual-ocaml/fonts/fira-sans-v8-latin-regular.woff2') format('woff2'), /* Super Modern Browsers */
-       url('/pub/docs/manual-ocaml/fonts/fira-sans-v8-latin-regular.woff') format('woff'), /* Modern Browsers */
-       url('/pub/docs/manual-ocaml/fonts/fira-sans-v8-latin-regular.ttf') format('truetype'), /* Safari, Android, iOS */
-       url('/pub/docs/manual-ocaml/fonts/fira-sans-v8-latin-regular.svg\#FiraSans') format('svg'); /* Legacy iOS */
+       url('fonts/fira-sans-v8-latin-regular.eot?\#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('fonts/fira-sans-v8-latin-regular.woff2') format('woff2'), /* Super Modern Browsers */
+       url('fonts/fira-sans-v8-latin-regular.woff') format('woff'), /* Modern Browsers */
+       url('fonts/fira-sans-v8-latin-regular.ttf') format('truetype'), /* Safari, Android, iOS */
```

Comment:
It is harmless to leave this in place; we won't be providing these other formats, but they could be available in the local installation, etc.

Reply:
Hmm that doesn't make sense to me (the user's local fonts surely would not be at this relative path? I would assume the name of the font, rather than a path, would be used in this case) but then, I know nothing about web fonts so I'm just going to trust you there.

Reply:
Yes, sorry, you are right! In any case it is harmless to leave it there.

Reply:
OK, let's just copy everything and be done with it.